src/components/ActionButton/index.stories.tsx

```diff
@@ -0,0 +1,72 @@
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Story, Meta } from '@storybook/react/types-6-0';
+import { Copy } from '../Icons';
+
+import ActionButton from './index';
+import { IActionButtonProps } from './interfaces';
+
+export default {
+  title: 'Action Button',
+  component: ActionButton,
+  argTypes: {
+    messageDuration: {
+      description: 'How long the tooltip remains on screen after click (ms).',
+      table: {
+        type: { summary: 'number' },
+        defaultValue: { summary: 1500 },
+      }
+    },
+    hoverContent: {
+      description: 'Infotip content to show on hover.',
+      control: 'text',
+      table: {
+        type: { summary: 'React.Node' },
+      }
+    },
+    clickContent: {
+      description: 'Infotip content to show on click.',
+      control: 'text',
+      table: {
+        type: { summary: 'React.Node' },
+      }
+    },
+    onClick: {
+      description: 'Action. Hover infotip shows on success.',
+      table: {
+        type: { summary: '() => boolean' },
+      }
+    },
+    buttonStyles: {
+      control: 'object',
+      description: 'CustomButton styles.',
+      table: {
+        type: { summary: 'ButtonProps' },
+      }
+    },
+    infotipProps: {
+      control: 'object',
+      description: 'Infotip properties.',
+      table: {
+        type: { summary: 'IInfotipProps' },
+      }
+    }
+  },
+} as Meta;
+
+const Template: Story<IActionButtonProps> = (args) => (
+    <ActionButton {...args} >
+      <Copy/>
+    </ActionButton>
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  onClick: () => true,
+  messageDuration: 1500,
+  clickContent: <span>Some action occurred.</span>,
+  hoverContent: <span>Hovering message.</span>,
+  infotipProps: {
```

Comment:
Is this alright? To reference another component props?

Reply:
I wouldn't -- just because if you end up removing the info-tip from the implementation you'll be forced to implement these props again.
I would suggest making them top level props on your component and passing them down to the infotip.